lnrpc/routerrpc/router.proto

```diff
@@ -373,8 +355,7 @@ message PairHistory {
     /// Minimum penalization amount (only applies to failed attempts).
     int64 min_penalize_amt_sat = 4 [json_name = "min_penalize_amt_sat"];
 
-    /// Estimation of success probability for this pair.
-    float success_prob = 5 [json_name = "success_prob"];
+    reserved 5;
```

Comment:
Discussed offline. We don't want probabilities for an implicit amount that are prone to misinterpretation.

Reply:
Yes, a follow up pr adds this info again, but in a separate rpc call that allows the caller the specify an amount.

The "assume" part was indeed light. The only assumption is that the probability applies to any amount greater than or equal to `min_penalize_amt`. This could also have been clarified with a comment. 

For the previously existing `other_success_prob` it was less clear to what amount it actually applied.

But bottom line is that with the changes in this and follow up prs, those assumptions don't hold anymore.